pkg/oc/bootstrap/docker/openshift/servicecatalog.go

```diff
@@ -180,6 +180,16 @@ func (h *Helper) InstallServiceCatalog(f *clientcmd.Factory, configDir, publicMa
 			},
 			"spec": map[string]interface{}{
 				"url": "https://apiserver.openshift-template-service-broker.svc:443/brokers/template.openshift.io",
+				"insecureSkipTLSVerify": true,
```

Comment:
@deads2k ultimately we presumably want this to go away too...  does the TSB have a cacert it should be providing to the SC (the SC's broker resource can accept one)?  Or since both components are running on the cluster i can just set this to false and the SC will trust the TSB's cert already?

Reply:
> does the TSB have a cacert it should be providing to the SC (the SC's broker resource can accept one)?

The service serving cert CA.  I'd suggest making this entire thing a template that takes that as a "standard" parameter from the installer, since the installer knows the value..  @sdodson @ewolinetz

Reply:
Oh, and for this pull, you actually have the value in cluster up since you create the service serving cert signer.

Reply:
While I agree in theory that's better, there's currently some timing issues with that.  If we create the broker resource before the TSB is actually up, the SC will immediately try to it it, fail, and not retry it for at least 5 minutes.  Meaning introducing significant lag between when the cluster is up, and when the templates show up in the web console.

I could create yet another resource/template file just for the broker resource object, but, ew.

Reply:
let's put this off to another PR for now.